src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -164,6 +164,11 @@
   <data name="MisplacedTransformFileWarning" xml:space="preserve">
     <value>The transform file '{0}' is outside the 'content' folder and hence will not be transformed during installation of this package. Move it into the 'content' folder.</value>
   </data>
+  <data name="NoRefOrLibFolderInPackage" xml:space="preserve">
+    <value>This package does not contain a lib/ or ref/ folder, and is therefore being treated as compatible for ALL
```

Comment:
> and is therefore being treated

> will therefore be treated

The problem will happen when you install a package, not at pack time.


> ALL = all. 

Avoid uppercase in user messages.